.github/workflows/tests.yaml

```diff
@@ -114,10 +114,10 @@ jobs:
                     "./scripts/py_matter_idl/matter_idl/xml_parser.py \
                       --no-print \
                       --log-level info \
-                      src/app/zap-templates/zcl/data-model/chip/global-attributes.xml \
-                      src/app/zap-templates/zcl/data-model/chip/access-control-definitions.xml \
                       src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml \
```

Comment:
please separate out the sort in a different PR. While desirable, we should not mix deltas.

In particular, I think global-attributes has to stay at the top for now.